### PR TITLE
initial checkin of auto versioning

### PR DESCRIPTION
**Pull request by RussTheAerialist**
Adding Versioneer support into the library to support integration testing across boundaries. (i.e. new version number per checkin).

### CODE REVIEW

**Comment by t8y8** [comment]
Just ran a test and it totally works:
```
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ git add test.test
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ git commit -m "1"
[merica 2f220e9] 1
 1 file changed, 1 insertion(+)
 create mode 100644 test.test
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ python -c "import tableauserverclient as TSC; print(TSC.__version__)"
0.2+60.g2f220e9
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ vim test.test
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ git add test.test
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ git commit -m "2"
[merica 0248c42] 2
 1 file changed, 1 insertion(+)
(tableau) TDOYLE-MAC1:server-client-python tdoyle$ python -c "import tableauserverclient as TSC; print(TSC.__version__)"
0.2+61.g0248c42
```

File: setup.cfg [hunk 1]
```diff
@@ -7,3 +7,16 @@ max_line_length = 120
...
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = tableauserverclient/_version.py
+versionfile_build = tableauserverclient/_version.py
+tag_prefix = v
+#parentdir_prefix =
```
Comment (t8y8) [inline setup.cfg]: nuke if we don't need
